nexus_constructor/component/component.py

```diff
@@ -73,6 +74,105 @@ def _transforms_are_equivalent(
     return transform_1.absolute_path == transform_2.absolute_path
 
 
+class TransformationsList(list):
+    def __init__(self, parent):
+        super().__init__()
+        self.parent_component = parent
+        self.link = LinkTransformation(self)
+
+    @property
+    def has_link(self):
```

Comment:
@matthew-d-jones This is the attribute I had to add in order to get the UI to work properly.

Reply:
Thanks, looks fine to me.